pom.xml

```diff
@@ -129,29 +137,40 @@
         <dependency>
             <groupId>javax.servlet</groupId>
             <artifactId>javax.servlet-api</artifactId>
-            <version>3.1.0</version>
         </dependency>
 
         <!-- graphql -->
         <dependency>
             <groupId>org.sangria-graphql</groupId>
             <artifactId>sangria_2.12</artifactId>
-            <version>1.4.2</version>
+            <version>${sangria.version}</version>
         </dependency>
         <dependency>
             <groupId>org.sangria-graphql</groupId>
             <artifactId>sangria-relay_2.12</artifactId>
-            <version>1.4.2</version>
+            <version>${sangria-relay.version}</version>
         </dependency>
         <dependency>
             <groupId>org.sangria-graphql</groupId>
             <artifactId>sangria-json4s-native_2.12</artifactId>
-            <version>1.0.0</version>
+            <version>${sangria-json4s-native.version}</version>
         </dependency>
         <dependency>
             <groupId>org.sangria-graphql</groupId>
             <artifactId>sangria-slowlog_2.12</artifactId>
-            <version>0.1.8</version>
+            <version>${sangria-slowlog.version}</version>
+        </dependency>
+
+        <!-- xml -->
+        <dependency>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
+            <version>${jakarta-xml-bind.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jaxb</groupId>
+            <artifactId>jaxb-runtime</artifactId>
+            <version>${jaxb-runtime.version}</version>
```

Comment:
why were these dependencies added?

Reply:
i have no clue how this happened, removed them.